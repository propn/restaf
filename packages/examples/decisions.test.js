let testFunctions = require("./testFunctions");
let testInfo;
beforeAll(async () => {
	testInfo = await require('./lib/setupAll')();
});
test("List repositories", async () => {
  let l = await testFunctions.modelRepoRoot(testInfo);
  expect(l).toMatchSnapshot();
});
test.only('Create CAS Destination', async () => {
	let l = await testFunctions.modelDestinationCas(testInfo);
	expect(l).toBe('done');
});
