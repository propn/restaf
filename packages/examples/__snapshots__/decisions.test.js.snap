// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List repositories 1`] = `
Immutable.Seq [
  "self",
  "models",
  "createModel",
  "createModelProject",
  "projects",
  "createProject",
  "repositories",
  "createRepository",
]
`;
